backend/ServiceLayer/ContactService.cs

```diff
@@ -32,8 +32,18 @@ public ContactService(CrmContext context, IMailService mailProvider) : base(cont
         public async Task SendDisclosure(long id)
         {
             Contact contact = await GetByIdAsync(id);
+            string gender = " geehrter Herr ";
+            if (contact.Gender == Contact.GenderTypes.FEMALE)
+            {
+                gender = " geehrte Frau ";
+            }
+            else if (contact.Gender == Contact.GenderTypes.DIVERS)
+            {
+                gender = " geehrt ";
```

Comment:
Bei Divers: geehrte/r  Herr oder Frau  statt geehrt

Reply:
![Screenshot 2020-09-13 194620](https://user-images.githubusercontent.com/60075524/93024874-d2c4dc00-f5f9-11ea-85f9-26a5089f468e.png)
so ist es aber in der tabelle explizit vermerkt